sklearn/mixture/_gaussian_mixture.py

```diff
@@ -728,13 +728,21 @@ def _initialize(self, X, resp):
         elif self.covariance_type == "full":
             self.precisions_cholesky_ = np.array(
                 [
-                    linalg.cholesky(prec_init, lower=True)
+                    np.flipud(
+                        np.fliplr(
+                            linalg.cholesky(np.flipud(np.fliplr(prec_init)), lower=True)
+                        )
+                    )
```

Comment:
Could you:
 - define a function (e.g. named `exchange`) for the composition of `np.flipud` and of `np.fliplf` and another function for the system resolution (which composes `exchange and `linalg.cholesky`) and reuse them?
  - inline the mathematical derivation given in https://github.com/scikit-learn/scikit-learn/issues/26415 in a comment, or at last link to this issue?

Reply:
The required permutation of matrix and the computation of `precision_cholesky` are now implemented by functions `_flipudlr` and `_compute_precision_cholesky_from_precision`, respectively. Mathematical derivation is added to the docstring as well.